cmd/server.go

```diff
@@ -129,6 +140,41 @@ func main() {
 		bufferManager = &buffer.OnDemandBufferManager{}
 	}
 
+	var tileCache cache.Cache
+	if dynamoTableName != "" && dynamoCacheRegion != "" {
```

Comment:
I'll probably remove this dynamo stuff, but I'm leaving it here as an example of how another cache might be configured.